wikimedia/wikipedia-ios
# Image recommendations - Add tooltips

**Phabricator:** https://phabricator.wikimedia.org/T359135

### Notes
This PR introduces a new component to the Components package for displaying a series of tooltips. The tooltip design itself is displayed via UIKit in `WKTooltipViewController`. I am also presenting the tooltips through a view controller protocol extension called `WKTooltipPresenting`. This extension is designed to handle the display and dismissal of the tooltips, so that the implementing view controller only needs to worry about constructing the tooltip view models.

As a part of this, I removed the full screen modal handling for iPad, to confirm tooltips are aligned properly. I think we should keep it removed for now to make the feature usable on iPad. We can fine-tune it later when we pick up https://phabricator.wikimedia.org/T360036.

### Test Steps
1. Fresh install the Staging app on device.
2. Go to the Image Recommendations feature.
3. Confirm tooltips are presented after the onboarding modal is dismissed according to the Figma design.
4. Leave the feature, then come back.
5. Confirm tooltips are not presented.
6. Tap the "Tutorial" button in the overflow menu.
7. Confirm tooltips are presented again.
